docs/administration/workers/index_[2018.2,).md

```diff
@@ -0,0 +1,62 @@
+---
+title: Workers
+description: Worker setup and configuration.
+position: 1400
+---
+
+When your deployment process runs a script or extracts a package without involving a Tentacle or SSH target, they process is performed by a worker (see [where steps run](#where-steps-run) below). By default the worker lives inside the Octopus Server process (the built-in worker), however you can also configure an [external worker](#external-worker). 
+
+In many cases running user provided scripts under the context of the Octopus Server is undesirable, so we have 
+
+## Built-in Worker
+
+Any time a script needs to be run (e.g. PowerShell or Calamari), Octopus Server would spawned a new process from it's process it's security context. In `2018.1.0` we introduced the ability to [change the security context](docs/administration/security/built-in-worker) under which those new processes spawned, adding a layer of security.
+
+:::hint
+Future versions of Octopus Server will bundle a worker with the server that is started automatically. It will be run under the user contect supplied through the `builtin-worker` command.
+:::
+
+## External Worker
+
+In `2018.2.0` we added the ability to nominate a Tentacle to perform the role of the worker. This Tentacle can run under a different user account and also on a different machine, adding extra layers of security. The built in worker is automatically disabled when an external worker is configured, preventing the Octopus Server from running any user provided scripts.
+
+:::hint
+Future versions of Octopus Server will expand on the worker concept, allowing you to create worker pools with multiple workers. Steps then can be configured to run on those pools.
```

Comment:
Perhaps it's worth adding a link to the spec here which we'll remove when shipped?